pkg/apis/velero/shared/data_move_operation_progress.go

```diff
@@ -24,6 +24,9 @@ type DataMoveOperationProgress struct {
 	// +optional
 	TotalBytes int64 `json:"totalBytes,omitempty"`
 
+	// +optional
+	IncrementalBytes int64 `json:"incrementalBytes,omitempty"`
```

Comment:
As discussed previous, we should not add this to Velero's progress as it is meaningless until backup completes.

Reply:
@Lyndon-Li the current PR doesn't actually fill this value in until the end, so even with the current implementation, it doesn't show up in status until DU is completed. So what you propose should be a relatively simple change -- just move the field to status directly.